src/pet.js

```diff
@@ -0,0 +1,77 @@
+const MINIMUM_AGE = 0;
+const MAXIMUM_AGE = 30;
+const MINIMUM_HUNGER = 0;
+const MAXIMUM_HUNGER = 10;
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0;
+
+class Pet {
+    constructor(name) {
+        this.name = name;
+        this.age = MINIMUM_AGE;
+        this.hunger = MINIMUM_HUNGER;
+        this.fitness = MAXIMUM_FITNESS;
+    };
+
+    get isAlive() {
+        return this.age < MAXIMUM_AGE && 
+               this.hunger < MAXIMUM_HUNGER && 
+               this.fitness > MINIMUM_FITNESS;
+    };
+
+    growUp() {
+        if(!this.isAlive) {
+            throw new Error('Your pet is no longer alive :(');
+        }else{
+            this.age += 1
+            this.hunger += 5;
+            this.fitness -= 3;
+        }   
+    };
+
+    walk() {
+        if(!this.isAlive) {
+            throw new Error('Your pet is no longer alive :(');
+        }else{
+            if((this.fitness + 4) <= MAXIMUM_FITNESS){
+                this.fitness += 4;
+            }else{
```

Comment:
but this one is important because the code in the if block does not return. Let me know if that does not make sense :)